.codesandbox/sandbox/package.json

```diff
@@ -8,7 +8,7 @@
     "build": "parcel build index.html"
   },
   "dependencies": {
-    "@inrupt/solid-client": "../../",
```

Comment:
Just out of curiosity - was this equivalent to "file:../../" (i.e. is the 'file:' prefix optional or something)...?

Reply:
It appears so - my first attempt at fixing this was actually prepending `file:`.